### ISSUE TITLE
Under tremendous heat/pressure, matter should vaporize/disappear

### ISSUE THREAD

**Issue by Waluer** [studio-minus/ppg-bugs-and-requests#2120]
**Feature description**
Simple as. Under extreme heat and pressure any object including inorganic and organic both vaporize or just disappear like how the decimator disassembles atoms.

**Feature purpose**
To make things more fun and reasonable, example wiring about 20 generators to a thruster have interesting results but is anticlimactic in the end because the object is pretty much indestructible. This would add more ways to kill creatures.

**Additional context**
This would be a lot easier than making every object destructible and having seperate debris sprites for all. For this you'd only need a smoke particle effect in terms of graphics.

**Comment by DindinYT37** [comment]
Great idea, but should very much be toggleable in my opinion. Since some people might want to keep their humans.

**Comment by TeslaCoil9001** [comment]
Humans and other organic matter could disintegrate into ash particles too!

**Comment by DindinYT37** [comment]
> Humans and another organic objects should disintegrate into ash particles too!

Did you even read the issue? "Under extreme heat and pressure any object including inorganic **and organic** both vaporize or just disappear like how the decimator disassembles atoms."

**Comment by Waluer** [comment]
> Great idea, but should very much be toggleable in my opinion. Since some people might want to keep their humans.

Also, one thing. Don't mistake the extreme pressure for the vanilla. You know how nukes propel objects even when they're like 10ft away from the ground zero? We don't want that. We want the object to disappear as soon as the pressure/heat wave hits so it is more realistic. I mean the force is so much that your skin and muscles tear away and bones shatter into millions of tiny pieces  because even if gravity was strong to hold you in position, matter might not be that durable and give loose. And the slider idea is very cool too.

**Comment by DindinYT37** [comment]
> > Great idea, but should very much be toggleable in my opinion. Since some people might want to keep their humans.
> 
> Also, one thing. Don't mistake the extreme pressure for the vanilla. You know how nukes propel objects even when they're like 10ft away from the ground zero? We don't want that. We want the object to disappear as soon as the pressure/heat wave hits so it is more realistic. I mean the force is so much that your skin and muscles tear away and bones shatter into millions of tiny pieces because even if gravity was strong to hold you in position, matter might not be that durable and give loose. And the slider idea is very cool too.

I get it, but it should still be toggleable, since people might want to keep their humans (and objects) non-vaporized.

**Comment by Waluer** [comment]
> > > Great idea, but should very much be toggleable in my opinion. Since some people might want to keep their humans.
> > 
> > 
> > Also, one thing. Don't mistake the extreme pressure for the vanilla. You know how nukes propel objects even when they're like 10ft away from the ground zero? We don't want that. We want the object to disappear as soon as the pressure/heat wave hits so it is more realistic. I mean the force is so much that your skin and muscles tear away and bones shatter into millions of tiny pieces because even if gravity was strong to hold you in position, matter might not be that durable and give loose. And the slider idea is very cool too.
> 
> I get it - but it should still be toggleable, since people might want to keep their humans (and objects) non-vaporized.

Yeah I agree, accidentally typed slider instead of toggleable. So if it ever get's added it should be toggleable.

**Comment by username90001** [comment]
> > Great idea, but should very much be toggleable in my opinion. Since some people might want to keep their humans.
> 
> Also, one thing. Don't mistake the extreme pressure for the vanilla. You know how nukes propel objects even when they're like 10ft away from the ground zero? We don't want that. We want the object to disappear as soon as the pressure/heat wave hits so it is more realistic. I mean the force is so much that your skin and muscles tear away and bones shatter into millions of tiny pieces because even if gravity was strong to hold you in position, matter might not be that durable and give loose. And the slider idea is very cool too.

but that makes recreating operation plumbob impossible
